src/image-button/image-button.style.tsx

```diff
@@ -21,29 +21,45 @@ export const Main = styled.button`
 
     img {
         width: 3.5rem;
-    }
-
-    ${MediaQuery.MaxWidth.tablet} {
-        padding: 1.5rem 0rem;
+        height: 3.5rem;
+        object-fit: contain;
     }
 
     ${MediaQuery.MaxWidth.mobileL} {
-        padding: 1rem 0rem;
+        padding: 0.5rem 0.25rem;
+        gap: 0.25rem;
     }
 
-    ${MediaQuery.MinWidth.tablet} {
-        :focus:active {
-            box-shadow: ${DesignToken.ElevationBoxShadow};
-            ${TextStyleHelper.getTextStyle("H4", "bold")}
-        }
+    ${(props) => {
+        if (props.$error) {
+            return css`
+                background: ${Color.Neutral[8]};
+                border: 1px solid ${Color.Validation.Red.Border};
 
-        @media (hover: hover) {
-            &:hover {
-                box-shadow: 0px 8px 16px -8px ${Color.Shadow.Elevation};
-                ${TextStyleHelper.getTextStyle("H4", "bold")}
-            }
+                :hover {
+                    box-shadow: 0 0 4px 1px ${Color.Shadow.Red};
+                }
+            `;
+        } else if (props.$selected) {
+            return css`
+                background: ${Color.Accent.Light[5]};
+                border: 1px solid ${Color.Primary};
+                color: ${Color.Primary};
+
+                :hover {
+                    box-shadow: 0 0 4px 1px ${Color.Shadow.Accent};
+                }
+            `;
+        } else {
+            return css`
+                &:hover {
+                    border: 1px solid ${Color.Accent.Light[1]};
+                    box-shadow: 0 0 4px 1px ${Color.Shadow.Accent};
+                    ${TextStyleHelper.getTextStyle("H4", "bold")}
+                }
+            `;
         }
-    }
+    }}
 
     :disabled {
```

Comment:
set border to transparent here as well

Reply:
should have specified. this is intended for the disabled style to remove the box-shadow on hover

Reply:
ah, my bad, didn't check if disabled state had any hover effects. fixed now.